pkg/config/config.go

```diff
@@ -88,6 +87,11 @@ func NewConfigForDelete(id string, fileName string, properties map[string]map[st
 	return newConfig(id, "", nil, filterProperties(id, properties), api, fileName)
```

Comment:
`NewConfigForDelete` seems to not be used anywhere.

Reply:
This is legacy config file. I would rather not touch it any more than necessary :laughing: 
I guess the plan would be, to get rid of all legacy files within the next few versions.